shared/unattended/Fedora-25.ks

```diff
@@ -1,12 +1,11 @@
-install
 KVM_TEST_MEDIUM
 GRAPHICAL_OR_TEXT
 lang en_US
 keyboard us
 network --bootproto dhcp --hostname atest-guest
 rootpw 123456
 firewall --enabled --ssh
-selinux --enforcing
+selinux --permissive
```

Comment:
Do you have a reason to switch to permissive? Unless you have a great reason to use permissive I'd suggest using enforcing to notify potential issues with selinux. We do want to make sure all usual operations work with selinux enabled and our customers are suppose to use selinux in enforcing mode.

Reply:
Really sorry for forgetting to change it to --enforcing here.